Commons/src/main/java/eu/neclab/ngsildbroker/commons/tools/HttpUtils.java

```diff
@@ -353,10 +362,37 @@ private static Uni<String> getReplyBody(List<String> acceptHeader, int endPoint,
 				return Uni.createFrom().failure(new ResponseException(ErrorType.NotAcceptable,
 						"Provided accept types " + acceptHeader + " are not supported"));
 		}
+
 		return Uni.createFrom().item(replyBody);
 
 	}
 
+	@SuppressWarnings("unchecked")
+	private static void conciseRepresentation(Object compacted, Map<?, ?> parent, String key) {
+		// First Situation : if compacted have array
+		if (compacted instanceof ArrayList<?> list) {
+			list.forEach(item -> {
+				if (item instanceof Map<?, ?> mapItem) {
+					conciseRepresentation(mapItem, null, "");
```

Comment:
Same as above

Reply:
because I am checking empty string in further execution